sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/LoginSteps.java

```diff
@@ -57,8 +57,8 @@ public LoginSteps(
         "I log in with the user",
         () -> {
           webDriverHelpers.accessWebSite(environmentUrl);
-          webDriverHelpers.fillInWebElement(LoginPage.USER_NAME_INPUT, "NatUser");
-          webDriverHelpers.fillInWebElement(LoginPage.USER_PASSWORD_INPUT, "NatUser38118");
+          webDriverHelpers.fillInWebElement(LoginPage.USER_NAME_INPUT, "SurvOff");
```

Comment:
Why change from NatUser to SurvOff?

Reply:
Good question, I couldn't make tasks for the Nat user, possible bug, need to address this question.

![Screenshot 2021-05-20 at 11 44 55](https://user-images.githubusercontent.com/9676903/118948284-cd761900-b960-11eb-811c-9991853cf41c.png)

Reply:
Makes sense, guessing NatUser cannot assign task to himself or same level users only lower level ones, but other tests maybe wont run with SurfOff since he might not have access to something else. Maybe make multiple LoginSteps with different users ?

Reply:
Good point, I will implement in this PR this.